extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -24,8 +26,17 @@ const FONTS_TO_LOAD = [
   },
 ];
 
-const renderShoppingTagTemplate = (tagData) => (
-  <div class="amp-story-shopping-tag-inner">
+/**
+ * @param {!ShoppingConfigDataDef} tagData
+ * @param {function(Event)} onClick
```

Comment:
Look carefully at where the function is located. It is not within the class scope, so it would use the global "this" instead of the this that we want to use, thus giving us an error. This is why I pass it in as a function parameter.